### PR TITLE
Trustore section isn't covered

### PR DESCRIPTION
**Pull request by cwiechmann**
When the IdP is configured with the following option (using an HTTPS-Metadata-URL):
```xml
        <SamlIdentityProvider
                entityId="https://keycloak.int.acme.com:8443/auth/realms/Axway"
                metadataUrl="https://keycloak.int.acme.com:8443/auth/realms/Axway/protocol/saml/descriptor"
                userNameAttribute="urn:oid:2.5.4.42"
                verifyAssertionExpiration="true">
```
The metadataUrl points to an HTTPS-Endpoint which is used by the API-Manager at startup to download the required Metadata-Information. In order to trust the certificate of this endpoint, a truststore must be provided. This is actually not covered by the documentation.

The keycloak example we ship with the product contains such an example:
```xml
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<SSOConfiguration>
    <CertificateValidation
        pathValidation="false"
        enableRevocation="false"
        trustStorePath="conf/trustStore.jks"
        trustStorePassword="Change this to the truststore passphrase for trustStore.jks"
         >
    </CertificateValidation>
```
And we should document it.

### CODE REVIEW

**Comment by cwiechmann** [comment]
@andreamussap, I'm planning to record a video how to setup SSO for the API-Manager as many customers are struggling with that topic. Once the video is published on our official Axway API-Management Youtube-Channel I would like to link it from the documentation. 

What do you think about that plan?

**Comment by cwiechmann** [comment]
I also added a Troubleshoot topic for that problem.

File: content/en/docs/apim_administration/apimgr_admin/sso/saml_sso_config.md [hunk 1]
```diff
@@ -17,9 +17,17 @@ Before you can start to configure API Manager SSO:
...
+A truststore that contains the required IdP server certificates and referenced in `service-provider.xml`.
+
+* When specifing the IdP by URL this truststore is used to validate the IdP server certificate to download the required Metadata-Information, see [Specify the IdP by URL](#Specify2).
+* Located in your API Gateway instance folder (for example, `INSTALL_DIR/apigateway/groups/group-2/instance-1/conf`).
+* Add the server certicates into this trustore.
+
 ### A keystore (for example `sso.jks`)
 
-This file is the truststore generated by an administrator and referenced in `service-provider.xml`.
+This file is the keystore generated by an administrator and referenced in `service-provider.xml`.
```
Comment (cwiechmann) [inline content/en/docs/apim_administration/apimgr_admin/sso/saml_sso_config.md]: I changed this from truststore to keystore, as the primary usage is to sign SAML-Request and validate responses. Not to trust something.

**Comment by andreamussap** [comment]
Hi @cwiechmann  _RE link video from the documentation_ -> This is very good ! You can embed the video the same way we did in [APIC concepts page](https://axway-open-docs.netlify.com/docs/central/fundamental_concepts/), e.g.:

```
Watch this short video to learn more about API proxies.

{{< youtube ja9HPLlzcn4 >}}
```

The **ja9HPLlzcn4** is the part of the YouTube video URL after **v=** --> `https://www.youtube.com/watch?v=ja9HPLlzcn4`

**Comment by andreamussap** [comment]
@cwiechmann I've reviewed your changes. Please take a last look and let me know if you're are happy with the updates, so I can merge this to production.

You can [have a look at the diff](https://github.com/Axway/axway-open-docs/pull/201/commits/88683c8ffa9691651f2a4d108714f74a18eca29d) to see what I changed, and see direct links to previews:

https://deploy-preview-201--axway-open-docs.netlify.com/docs/apim_administration/apimgr_admin/sso/saml_sso_config/#configuration-files

https://deploy-preview-201--axway-open-docs.netlify.com/docs/apim_administration/apimgr_admin/sso/c_saml_trouble/#error-retrieving-metadata

**Comment by cwiechmann** [comment]
Thanks for reviewing it @andreamussap. I'm happy with the changes you made. Feel free to merge it.

**Comment by andreamussap** [comment]
### Thank you
We really appreciate your contribution and thank you for helping us to improve this documentation!
 
### Where to find the updated documentation
An Axway technical writer has reviewed and merged your PR, which is now available on the following sites:
Netlify (staging): https://axway-open-docs.netlify.com/docs/
Zoomin (production): https://docs.axway.com/bundle/axway-open-docs/
 
### Send us your feedback
If you have a moment, please complete the following 2 minute survey to give us your feedback on this process and help us make it even better:
https://www.surveymonkey.com/r/PJ5HQFD

**Pull request merged**